lib/compat/wordpress-6.3/class-gutenberg-classic-to-block-menu-converter.php

```diff
@@ -31,7 +31,7 @@ public static function convert( $menu ) {
 		$menu_items = wp_get_nav_menu_items( $menu->term_id, array( 'update_post_term_cache' => false ) );
 
 		if ( empty( $menu_items ) ) {
-			return array();
+			return '';
```

Comment:
Should this return a WP_error object?

Reply:
It's also got me thinking. Any changes here will possible effect the fallback handling code so we'd need to check there.

I'd hope if the tests aren't failing it's all ok but it's entirely possible there isn't 100% coverage 🤔

Reply:
Actually I don't think this should be an error. That's because we have converted the menu but it was just empty. That doesn't make it invalid. The conversion was a success.

We're just exiting early to avoid running the "to_blocks" code.

Later on we throw an error if the conversion to blocks fails because that's a true "error".